src/routing/index.tsx

```diff
@@ -55,39 +83,36 @@ const getRoutes = () => {
     //     }
     // );
 
-    routes.push(
-        {
-            id: 'dokan-withdraw',
-            title: __( 'Withdraw', 'dokan-lite' ),
-            element: <Withdraw/>,
-            path: '/withdraw',
-            exact: true,
-            order: 10,
-        }
-    );
+    routes.push( {
+        id: 'dokan-withdraw',
+        title: __( 'Withdraw', 'dokan-lite' ),
+        element: <Withdraw />,
+        path: '/withdraw',
+        exact: true,
+        order: 10,
+    } );
```

Comment:
- Extracts the routes to a separate file like `routing/route.tsx` which will be updated frequently for new/update routes.
- Import routes here so that we do not need to modify the `index.tsx` file frequently.